Gemfile.lock

```diff
@@ -52,7 +52,7 @@ GEM
       i18n (>= 0.6.4, <= 2)
     nokogiri (1.8.5)
       mini_portile2 (~> 2.3.0)
-    pagerduty (2.0.0)
+    pagerduty (2.1.2)
```

Comment:
### [pagerduty](http://github.com/envato/pagerduty)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.0.0 → 2.1.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **74** commits. See the full changes on [the compare page](https://github.com/envato/pagerduty/compare/v2.0.0...v2.1.2).

These are the first 10 commits:
- (9c6a970) [Specs break with RSpec 3.x, limiting to 2.x.](https://github.com/envato/pagerduty/commit/9c6a9701abf11b3c57844555d8275cd331638a91)
- (f8495d2) [fix deprecations for rspec 2.99.x](https://github.com/envato/pagerduty/commit/f8495d2a380d74fffda08eaba1395fbdf656b828)
- (74a92ee) [How to inspect HTTP error responses](https://github.com/envato/pagerduty/commit/74a92ee2289f140f51be2d791b6ef1c24632cf7a)
- (8655957) [we don't encapsulate HTTP errors](https://github.com/envato/pagerduty/commit/8655957cea2896214cd531eb8bac9e64b9116139)
- (5bf2a97) [Add a license badge](https://github.com/envato/pagerduty/commit/5bf2a979df3efa37a625c11518641d4bfc08cef4)
- (dc34090) [Don't shadow `send` method](https://github.com/envato/pagerduty/commit/dc3409056aee7ba66e0edec0277c5d2a516c2b83)
- (c5e442e) [Update Travis rvm version testing](https://github.com/envato/pagerduty/commit/c5e442ea5605a5ef05b22c3f8e1887cd25fbcd8d)
- (66906b2) [Merge pull request #29 from envato/dont_shadow_send](https://github.com/envato/pagerduty/commit/66906b266f82fafdce673420fb5358da03793257)
- (f7852ec) [Merge pull request #31 from shortdudey123/update_travis_ruby…](https://github.com/envato/pagerduty/commit/f7852ec999326b8f016249c2d5952601b9c6ac6c)
- (af401af) [Remove rspec 2.0 version pin](https://github.com/envato/pagerduty/commit/af401afc7f2a166f29c93c0f1868195c27ddf237)


</details>